src/riot-rs-coap/src/udp_nal/util.rs

```diff
@@ -7,36 +7,34 @@ pub(super) fn sockaddr_nal2smol(sockaddr: nal::SocketAddr) -> Result<IpEndpoint,
     match sockaddr {
         #[allow(unused)]
         nal::SocketAddr::V4(sockaddr) => {
-            #[cfg(feature = "proto-ipv4")]
+            // #[cfg(feature = "proto-ipv4")]
```

Comment:
This whole module is maturing a bit more here and then is to be PR'd in embassy-net, where those cfgs are required. They can currently not be active because this crate doesn't have the proto-ipvX features: They'd get out of sync with those on embassy-net all the time (which won't happen in the upstreamed version), so currently this crate just enables v4 and v6 on embassy-net to ensure all is in sync.